server/rak.js

```diff
@@ -13,7 +13,13 @@ const EVENT_TYPE = {
   BUTTON_PRESS_2: 66, // ASCII for 'B'
   BUTTON_PRESS_3: 67, // ASCII for 'C'
   BUTTON_PRESS_4: 68, // ASCII for 'D'
+
   HEARTBEAT: 72, // ASCII for 'H'
+
+  BUTTON_PRESS_5: 49, // ASCII for 1
+  BUTTON_PRESS_6: 50, // ASCII for 2
+  BUTTON_PRESS_7: 51, // ASCII for 3
+  BUTTON_PRESS_8: 52, // ASCII for 4
```

Comment:
hmm I think these should still be named `BUTTON_PRESS_1 ... BUTTON_PRESS_4` for clarity, and then the v1 buttons should be renamed to `BUTTON_PRESS_A ... BUTTON_PRESS_D`. What do you think?

Reply:
Yes I think that makes sense! Made this change.